english/slap-it-hard/move/move.md

```diff
@@ -0,0 +1,15 @@
+The Marshmallow creature has a look of determination (erm, if you can call a cute fluffy face that. It has more of a Chow Chow face, you know, those super kawaii puffy doggies that just melt your heart when they waddle around or munch on their little biscuits. :3 Yeah, I've always wanted one of those, but the fur...man, how does one deal with so much shedding?! Though, you will have a puff cloud that loves you beyond reason, becoming your best friend that goes with you everywhere and anywhere, heck yeah! Yeah...Ahem, I digress.) as it lunges toward you.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.